### PR TITLE
Response tuning

### PR DESCRIPTION
**Pull request by shairozan**

### CODE REVIEW

File: job/delete.go [hunk 1]
```diff
@@ -89,6 +90,9 @@ func generallyDeleteIndividual(w http.ResponseWriter, r *http.Request, routePara
 		},
 	}
 
+	log.Info(outputs)
```
Comment (dpastoor) [inline job/delete.go]: more appropriate to debug/trace this?

File: scripts/init.d/chokhmah.sh [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+### END INIT INFO
+
+# Author: Darrell Breeden <darrellb@metrumrg.com>
+
+# Assumes Binary is placed in /usr/local/bin with all other custom binaries for opensips
+
+# Do NOT "set -e"
+
+# PATH should only include /usr/* if it runs after the mountnfs.sh script
+PATH=/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin
```
Comment (dpastoor) [inline scripts/init.d/chokhmah.sh]: is this the path that the chokhmah daemon would see? could this be a problem for finding qsub/qstat?

File: .goreleaser.yml [hunk 3]
```diff
@@ -33,9 +33,8 @@ nfpms:
       - deb
       - rpm
 
-    # Packages your package depends on.
+    # Packages your package depends on. Removing refeerence to SystemD since we're also including INITD
```
Comment (dpastoor) [inline .goreleaser.yml]: refeerence --> reference

File: scripts/init.d/chokhmah.sh (hunk 2 above)
Comment (shairozan) [reply scripts/init.d/chokhmah.sh]: No this path is purely for the init script itself. The service loads an ENV file that contains the path required for qstat etc.

File: job/delete.go (hunk 1 above)
Comment (shairozan) [reply job/delete.go]: Not even needed. Can remove it as it was used for debugging an issue for extraneous outputs from the CLI :)

File: .goreleaser.yml (hunk 3 above)
Comment (shairozan) [reply .goreleaser.yml]: doh! fixed :-D

**Pull request merged**